src/dist/replication/lib/mutation_log.cpp

```diff
@@ -190,7 +189,7 @@ void mutation_log_shared::write_pending_mutations(bool release_lock)
             // here we use _is_writing instead of _issued_write.expired() to check writing done,
             // because the following callbacks may run before "block" released, which may cause
             // the next init_prepare() not starting the write.
-            _is_writing = false;
+            _is_writing.store(false, std::memory_order_release);
```

Comment:
If I understand correctly, we should us acquire in this case as we don't want to the latter callback ops to be scheduled first. release seems unnecessary for me.